### PR TITLE
Add readme notes for running tests manually

### PR DESCRIPTION
**Pull request by zachgoldstein**
Closes: https://github.com/airbrake/airbrake-python/issues/47

### CODE REVIEW

**Comment by phumpal** [comment]
If I follow these instructions here is the output for `python setup.py test`

```
➜  airbrake-python git:(chore/add-readme-test-notes) ✗ python setup.py test
running test
running egg_info
writing requirements to airbrake.egg-info/requires.txt
writing airbrake.egg-info/PKG-INFO
writing top-level names to airbrake.egg-info/top_level.txt
writing dependency_links to airbrake.egg-info/dependency_links.txt
reading manifest file 'airbrake.egg-info/SOURCES.txt'
writing manifest file 'airbrake.egg-info/SOURCES.txt'
running build_ext
Traceback (most recent call last):
  File "setup.py", line 86, in <module>
    setup(**package_attributes)
  File "/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/distutils/core.py", line 151, in setup
    dist.run_commands()
  File "/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/distutils/dist.py", line 953, in run_commands
    self.run_command(cmd)
  File "/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/distutils/dist.py", line 972, in run_command
    cmd_obj.run()
  File "/Users/phumpal/Documents/airbrake/notifiers/airbrake-python/fuckaroo/lib/python2.7/site-packages/setuptools/command/test.py", line 211, in run
    self.run_tests()
  File "/Users/phumpal/Documents/airbrake/notifiers/airbrake-python/fuckaroo/lib/python2.7/site-packages/setuptools/command/test.py", line 234, in run_tests
    **exit_kwarg
  File "/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/main.py", line 94, in __init__
    self.parseArgs(argv)
  File "/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/main.py", line 149, in parseArgs
    self.createTests()
  File "/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/main.py", line 158, in createTests
    self.module)
  File "/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/loader.py", line 130, in loadTestsFromNames
    suites = [self.loadTestsFromName(name, module) for name in names]
  File "/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/loader.py", line 91, in loadTestsFromName
    module = __import__('.'.join(parts_copy))
ImportError: No module named tests
```

This occurs on  `Python 2.7.13`

**Comment by zachgoldstein** [comment]
@phumpal PTAL

File: README.md [hunk 1]
```diff
@@ -105,6 +105,32 @@ def bake(**goods):
         logger.error("No temperature defined!", extra=goods)
 ```
 
+#### Running Tests Manually
+Create your environment and install the test requirements
+```
+virtualenv venv 
+source venv/bin/activate
+python setup.py install
```
Comment (samstav) [inline README.md]: I believe `pip install .` or `pip install --editable .` is preferred.

http://stackoverflow.com/a/15731459/1547030

File: README.md [hunk 2]
```diff
@@ -105,6 +105,32 @@ def bake(**goods):
...
+python setup.py install 
+python setup.py test
+```
+
+To run via nose (unit/integration tests):
+```
+source venv/bin/activate
+pip install -r ./test-requirements.txt
+source venv/bin/activate  # to get 
+nosetest
```
Comment (samstav) [inline README.md]: `nosetests` with an s

Comment (zachgoldstein) [inline README.md]: 7oztu

**Comment by phumpal** [comment]
@zachgoldstein works for me now, thanks!

**Pull request merged**